package.json

```diff
@@ -12,148 +18,48 @@
     "node": ">=10.14.1"
   },
   "scripts": {
-    "build": "npx lerna run --stream build -- -s",
-    "clean:all": "npx lerna run clean && npx lerna clean --yes && rimraf ./node_modules",
-    "clean:dist": "npx lerna run clean",
+    "build": "yarn workspaces run build",
+    "build:dev": "yarn workspaces run build:dev",
+    "clean:all": "yarn workspaces run clean && rimraf ./node_modules",
+    "clean:dist": "yarn workspaces run clean",
     "coveralls": "cat ./coverage/lcov.info | coveralls",
     "danger": "danger-ci",
     "lint": "eslint '@(packages|scripts)/**/{*.js,package.json}' --ignore-pattern node_modules --ignore-pattern storybook-dist",
-    "now-build": "cp packages/venia-concept/.env.dist packages/venia-concept/.env && npm run -s build",
-    "now-start": "npm run -s stage:venia",
-    "prepare": "node ./scripts/update_repo_environment.js && npx lerna bootstrap --hoist",
+    "now-build": "cp packages/venia-concept/.env.dist packages/venia-concept/.env && yarn run build",
+    "now-start": "yarn run stage:venia",
     "prettier": "prettier --write '@(packages|scripts)/**/*.@(js|css)' '*.js'",
     "prettier:validate": "prettier-check '@(packages|scripts)/**/*.@(js|css)' '*.js'",
     "prettier:check": "prettier --list-different '@(packages|scripts)/**/*.@(js|css)' '*.js'",
-    "stage:venia": "cd packages/venia-concept && npm start; cd - >/dev/null",
-    "stats:venia": "cd packages/venia-concept && npm run build:stats && webpack-bundle-analyzer dist/build-stats.json",
-    "storybook:venia": "cd packages/venia-concept && npm run storybook",
+    "stage:venia": "cd packages/venia-concept && yarn run start; cd - >/dev/null",
+    "stats:venia": "cd packages/venia-concept && yarn run build:stats && webpack-bundle-analyzer dist/build-stats.json",
+    "storybook:venia": "cd packages/venia-concept && yarn run storybook",
     "test": "jest",
     "test:ci": "jest --no-cache -i --json --outputFile=test-results.json",
     "test:debug": "node --inspect-brk node_modules/.bin/jest -i --no-cache",
     "test:dev": "jest --watch",
-    "validate-queries": "npx lerna run validate-queries -- -s",
+    "validate-queries": "cd packages/venia-concept && yarn run validate-queries",
     "watch:all": "node scripts/watch-all.js",
-    "watch:buildpack": "cd packages/pwa-buildpack && npm run -s watch; cd - >/dev/null",
-    "watch:peregrine": "cd packages/peregrine && npm run -s watch; cd - >/dev/null",
-    "watch:venia": "cd packages/venia-concept && npm run -s watch; cd - >/dev/null"
-  },
-  "husky": {
```

Comment:
i know you aren't a huge fan of git hooks but can we leave this in?

Reply:
Yes, I'll add it back as a very last step when this PR is approved.

Reply:
still would like this not to be removed.